lib/BloqadeQMC/src/ising/Rydberg.jl

```diff
@@ -69,86 +69,37 @@ function make_prob_vector(H::Type{<:AbstractRydberg}, V::UpperTriangular{T}, Ω:
     return ops, p, energy_shift
 end
 
+total_hx(H::Rydberg)::Float64 = sum(H.Ω) / 2
+haslongitudinalfield(H::AbstractRydberg) = !iszero(H.δ)
```

Comment:
do not use concatenated words, use snake case.

Reply:
This might mess up the internal structure of the QMC code at the moment. I believe it would probably be better to start to rename things in when the refactoring occurs.

Reply:
Yup